include/accessor.h

```diff
@@ -604,8 +605,9 @@ class local_accessor {
 
   private:
 	sycl_accessor m_sycl_acc;
-	range<Dims> m_allocation_size;
+	CELERITY_DETAIL_NO_UNIQUE_ADDRESS range<Dims> m_allocation_size;
 	cl::sycl::handler* const* m_eventual_sycl_cgh = nullptr;
+	// TODO after multi-pass removal: verify that sizeof(celerity::local_accessor) == sizeof(sycl::local_accessor) in accessor_tests
```

Comment:
You could add a `[!shouldfail]` test for this now!

Reply:
Done.